docs/tutorial.rst

```diff
@@ -10,8 +12,16 @@ This tutorial will walk you through using ELECTRIC to analyze the electric field
         - You are able to download or clone a directory from git.
         - You are familiar with bash scripts.
 
+To begin thee tutorial, first make sure you have your `electric` environment activated:
```

Comment:
Typo: "thee"

Reply:
Thanks for the update.  Other than the small typo, it looks good to me.

Reply:
Thanks!